docs/diagrams/RunningLowSequenceDiagram.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+!include style.puml
+
+participant ":RunningLowCommand" as RunningLowCommand
+participant "foodCategory :FoodCategory" as FoodCategory
+participant "fridge:Fridge" as Fridge
+participant "food :Food" as Food
+
+-> RunningLowCommand: execute()
+activate RunningLowCommand
+
+loop foodCategory in FoodCategory
+    RunningLowCommand -> FoodCategory: updateMessage()
+    activate FoodCategory
+    FoodCategory -> FoodCategory: getMinimumQuantity()
```

Comment:
Am I seeing things, or is the self-invocation here not spawning a new activation bar? Is that supposed to be correct?